.circleci/config.yml

```diff
@@ -31,3 +31,8 @@ jobs:
             - store_artifacts:
                 path: pi-image.img.xz
                 destination: pi-image.img.xz
+            - store_artifacts:
+                path: sb-debs
```

Comment:
I believe the fix to @kierdavis's comment on https://github.com/sourcebots/pi-image/pull/12#issuecomment-361512280, adding a `destination` key will put things in the root of the artifacts, which feels much nicer!

Reply:
aaafa2a